src/node_input_factory/node_input_classes.py

```diff
@@ -57,11 +57,17 @@ def __init__(self, location: str, name: str, node_purpose: str):
 
 class EngineNodeInput(NodeInput):
     # Enum 3
-    def __init__(self, set_engine: bool, name: str, node_purpose: str):
+    def __init__(self, set_engine: int, name: str, node_purpose: str):
```

Comment:
Waarom is dit een int geworden? Een motorblok kan maar aan of uit staan toch? Of moeten we een "motor" van maken, zoals op de smarterdam. Met motor bedoel ik dan "iets wat een wiel voor/achteruit draait"

Reply:
Je hebt gelijk het kan beter op bool staan want een motorblok kan maar aan of uit staan inderdaad ik verander het terug naar bool